packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -71,18 +71,80 @@ import { watchTransactionForConfirmations } from './utils/watch_transaction_for_
 import { Web3EthExecutionAPI } from './web3_eth_execution_api';
 import { NUMBER_DATA_FORMAT } from './constants';
 
+/**
+ *
```

Comment:
I think we should add some note in documentation like:
All these functions are available in web3Eth class and normal users should use from there. These RPC wrapper functions are used internally in web3Eth class and are also available for advance users of lib if they want.

Reply:
There's no single place to add that disclaimer, but we did talk about linking to the `Web3Eth` methods for each `rpc_method_wrapper` method

Reply:
there are multiple rpc functions, you are right, but we can add this note at top (may be before first wrapper function only or at top section?)  like one of:
![image](https://user-images.githubusercontent.com/86780488/181477266-b6f581b4-e920-452d-a388-dd7a3befb78c.png)
https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
Made an [issue](https://github.com/ChainSafe/web3.js/issues/5295) to track this